src/components/ScrollLock/ScrollLock.ts

```diff
@@ -0,0 +1,21 @@
+import {useEffect} from 'react';
+import {useScrollLockManager} from '../../utilities/scroll-lock-manager';
+import './ScrollLock.scss';
+
+interface ScrollLockProps {}
+
+// Even though this has no args, reference ScrollLockProps so the prop explorer
```

Comment:
Will this show up in the style guide without a `Props` interface?

Reply:
On second thought, looks like we don't have a readme and this is just an internal component.

Reply:
Per my proposal in #1959 we now name Props after the component - see ScrollLockProps above this.

The styleguide was updated to handle this in https://github.com/Shopify/polaris-styleguide/pull/2932